File: assets/src/pet.cs
// Pet sample

public abstract class Pet
{
    public abstract string GetAngry();

    public static void Main(string[] args)
    {
        Pet pet = new Cat();
        System.Console.WriteLine(pet.GetAngry());
    }
}

public class Cat : Pet
{
    public override string GetAngry()
    {
        return @"ネコがひっかく";
    }
}

public class Dog : Pet
{
    public override string GetAngry()
    {
        return @"イヌがほえる";
    } 
}

File: assets/src/pet.py
# Pet sample

class Pet():
    def getAngry(self):
         pass

class Cat(Pet):
    def getAngry(self):
        return 'ネコがひっかく'

class Dog(Pet):
    def getAngry(self):
        return 'イヌがほえる'

pet = Cat()
print(pet.getAngry())
pet = Dog()
print(pet.getAngry())
